internal/config/config.go

```diff
@@ -511,3 +519,28 @@ func CopyHeader() bool {
 
 	return false
 }
+
+// Function for reading the ega key in []byte
+func GetJwtKey(jwtpubkeypath string, jwtKeys map[string][]byte) error {
```

Comment:
i think we can have `/jwk` endpoint and serve our key there so it creates less hassle when changing the function for checking jwt signature.

Reply:
Yes, that makes sense actually. I might need to leave this function as a second option though, at least until we create an endpoint for serving the key

Reply:
It's not in https://www.iana.org/assignments/well-known-uris/well-known-uris.xhtml, but `.well-known/jwks.json` would be a good path for that, see e.g. https://auth0.com/docs/secure/tokens/json-web-tokens/json-web-key-sets.